src/plugins/hetero/CMakeLists.txt

```diff
@@ -8,20 +8,24 @@ endif()
 
 set (TARGET_NAME "openvino_hetero_plugin")
 
-file(GLOB SOURCES ${CMAKE_CURRENT_SOURCE_DIR}/*.cpp)
-file(GLOB_RECURSE HEADERS ${CMAKE_CURRENT_SOURCE_DIR}/*.hpp)
+file(GLOB SOURCES ${CMAKE_CURRENT_SOURCE_DIR}/src/*.cpp)
+file(GLOB HEADERS ${CMAKE_CURRENT_SOURCE_DIR}/src/*.hpp)
 
 ov_add_plugin(NAME ${TARGET_NAME}
               DEVICE_NAME "HETERO"
               PSEUDO_DEVICE
               SOURCES ${SOURCES} ${HEADERS}
-              VERSION_DEFINES_FOR plugin.cpp
+              VERSION_DEFINES_FOR src/plugin.cpp
               ADD_CLANG_FORMAT)
 
 ie_faster_build(${TARGET_NAME}
     UNITY
 )
 
+target_include_directories(${TARGET_NAME} PRIVATE
+    "${CMAKE_CURRENT_SOURCE_DIR}/include/"
```

Comment:
why do we need includes for HETERO?

Reply:
Such header can be stored inside src folder IMO
Include folders are needed for external (relative to this component) users.

CC @ilyachur what do you think?

Reply:
> Such header can be stored inside src folder IMO Include folders are needed for external (relative to this component) users.
> 
> CC @ilyachur what do you think?

This include is already inside src: `src/plugins/hetero/include/hetero`

We have similar situation in some other plugins:
`src/plugins/template/include/template`
`src/plugins/intel_gpu/include/intel_gpu`

Reply:
> src/plugins/template/include/template

It's a public API of template plugin

> src/plugins/intel_gpu/include/intel_gpu

It has different folders structure because of plugin complexity..

Reply:
Decided to remove `include` folder